packages/ui/Toggleable/Toggleable.js

```diff
@@ -204,7 +212,7 @@ const ToggleableHOC = hoc(defaultConfig, (config, Wrapped) => {
 
 		handle = handle.bind(this)
 
-		forwardWithState = (evName) => (ev, props) => forward(evName, {[prop]: !this.state.active}, props)
+		forwardWithState = (evName) => (ev, props) => forward(evName, {...pick(eventProps, props), [prop]: !this.state.active}, props)
```

Comment:
If `eventProps` is somehow undefined/null, does `pick` fall over? Probably an over-engineered edge case, but I'm curious.